core/pom.xml

```diff
@@ -304,6 +311,11 @@
       <groupId>org.geotools</groupId>
       <artifactId>gt-geojson</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.jayway.jsonpath</groupId>
+      <artifactId>json-path</artifactId>
+      <version>2.4.0</version>
```

Comment:
I see `json-path` is used in `services` also. Please move the dependency with the version in both modules to the root pom and remove the version in the modules.

Reply:
moved